blog.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name= "viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <link rel="stylesheet" type="text/css" href="styles/normalize.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Lora:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <title>Adopt a Dog - Blog</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-top">
+        <div class="logo-bundle">
+          <img class="header-logo" src="images/adoptadog-logo.svg" alt="Adopt a Dog Logo">
+          <p>Seattle's premiere dog adoption agency</p>
+        </div>
+        <div class="shopping-cart">
+          <img class="shopping-cart-icon" src="images/cart-icon.svg" alt="Shopping cart icon">
+          <p>$1234.56</p>
+          <button class="cart-button" type="button">Checkout</button>
+        </div>
+      </div>
+      <div>
+        <ul class="header-nav">
+          <li><a href="index.html">Home</a></li>
+          <li><a href="our-dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </div>
+    </header>
+    <main>
+      <h1>Adoptadog Blog</h1>
+      <article>
```

Comment:
You'll want to create a CSS class to style these articles. So the text is beside the image. See PDF for example.